.gitignore

```diff
@@ -1,4 +1,10 @@
 __pycache__/
+*.pyc
+*.o
+CMakeFiles/*
+CMakeCache.txt
+Makefile
+cmake_install.cmake
```

Comment:
Our build system always builds in the directory `builds` which is already ignored (in line 5 of the old file). What is the reason to also ignore the build artifacts directly?

Reply:
At the time I coded this, Fast Downward was not on Git yet. It was Mercurial, and I had Gitified it myself.
For sure now I trust your .gitignore more. I will remove the commit.